admin/includes/stylesheet.css

```diff
@@ -835,6 +835,16 @@ input.faint {
 
 /* End of css buttons */
 
+/* Additions for help */
+.navbar {
+  margin-bottom: 8px;
+}
+.btn-help {
+  margin-right: 4px;
+  margin-bottom: 8px;
+  background-color: #5ce400; 

```

Comment:
I think this bright green is too strong. Something that matches existing colors is better IMO. Perhaps `#E7E6E0`

Reply:
OK.